src/test/encoding/readable.sh

```diff
@@ -4,23 +4,25 @@ dir=../ceph-object-corpus
 
 set -e
 
-tmp1=`mktemp /tmp/typ-XXXXXXXXX`
-tmp2=`mktemp /tmp/typ-XXXXXXXXX`
-
 failed=0
 numtests=0
+pids=""
 
 myversion=`./ceph-dencoder version`
+DEBUG=0
```

Comment:
@ErwanAliasr1 could you revise the title of your commit message with something like: "test/encoding/readable.sh: parallelize the tests" ?

so it is more specific?